Source/Core/Package/DPM.Core.Package.Installer.pas

```diff
@@ -466,7 +466,7 @@ function TPackageInstaller.DoInstallPackage(const cancellationToken : ICancellat
   projectReferences := TCollections.CreateList<TProjectReference>;
 
   projectReferences.AddRange(TEnumerable.Select<IPackageInfo, TProjectReference>(projectPackageInfos,
-    function(const info : IPackageInfo) : TProjectReference
+    function(info : IPackageInfo) : TProjectReference
```

Comment:
This is odd, what version of spring4d did you clone. You need 2.0-develop as the func is defined as 
````  Func<T, TResult> = reference to function(const arg: T): TResult;````